src/Apps/CollectorProfile/Routes/Saves/Components/ArtworkListsHeader.tsx

```diff
@@ -57,7 +64,19 @@ export const ArtworkListsHeader: FC<ArtworkListsHeaderProps> = ({
           alignItems={["stretch", "center"]}
         >
           <Text variant="sm-display" color="black60">
-            {t("collectorSaves.artworkListsHeader.curateYourList")}
+            {isPartnerOfferEnabled ? (
+              <ProgressiveOnboardingSaveTitle>
+                {t("collectorSaves.artworkListsHeader.curateYourList") +
+                  " " +
+                  t("collectorSaves.artworkListsHeader.connector") +
+                  " "}
+                <RouterLink to="TODO.com">
```

Comment:
Is there a Jira ticket to update this so that we don't forget?

Reply:
I added a question in this ticket so see if we can get it in before merge, I'll wait until tomorrow

Reply:
Got it. Thank you.

Reply:
I already created a ticket for it [EMI-1644](https://artsyproduct.atlassian.net/browse/EMI-1644) in case it will take a while